com.rabbitcompany.passky.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.rabbitcompany.passky
```

Comment:
```suggestion
app-id: com.rabbit_company.passky
```

`rabbitcompany.com` seems to really be unrelated, unlike `rabbit-company.com`

Reply:
I have received an error when I have used com.rabbit-company.passky, so I went without -

Reply:
that's why my suggestion had an **underscore** `_`. Which is the rule.